base/cvd/cuttlefish/host/commands/host_bugreport/main.cc

```diff
@@ -131,20 +137,30 @@ Result<void> CvdHostBugreportMain(int argc, char** argv) {
   CHECK(config) << "Unable to find the config";
 
   auto out_path = FLAGS_output.c_str();
-  std::unique_ptr<FILE, decltype(&fclose)> out(fopen(out_path, "wbe"), &fclose);
-  ZipWriter writer(out.get());
+  int err = 0;
+  std::unique_ptr<zip_t, decltype(&zip_discard)> archive(
+      zip_open(out_path, ZIP_CREATE | ZIP_TRUNCATE, &err), &zip_discard);
+  if (archive.get() == nullptr) {
```

Comment:
nit: I think you can simply ask `archive == nullptr`

Reply:
Changed to `!archive`.